### ISSUE TITLE
How to set up the dataset folder

### ISSUE THREAD

**Issue by Xlgd** [yangdb/RD-IOD#1]
Hello, I'm sorry to bother you. I would like to reproduce your experiment, could you please tell me how I can set up the dataset folder `data/`, my dataset is VOC2007 and it is set up according to [py-faster-rcnn](https://github.com/rbgirshick/py-faster-rcnn#beyond-the-demo-installation-for-training-and-testing-models).  I read the code and found that it will use folders such that `data/VOCdevkit2007-1/`, `data/VOCdevkit/2007-15/`... when doing incremental training. How do I divide the data to set up these folders? Thank you very much!

**Comment by yangdb** [comment]
"data/VOCdevkit2007-1/" includes the images and annotations of class "tv". "data/VOCdevkit/2007-15/" includes the images and annotations of the first 15 classes. You can divide the dataset according to .xml files.

**Comment by Xlgd** [comment]
Thank you very much for your answer! Regarding how to divide out "data/VOCdevkit2007-1/", I find the annotation and corresponding images containing "tv" from "trainval.txt" and put them into the "VOCdevkit2007-1/" folder as training set, and use the same method to find the annotation and images from "test.txt" and use them as test set, making sure that "VOCdevkit2007-1/" has the same structure as the original "VOCdevkit2007/". Do you think this is the correct way to divide the data set? And do you have any suggestions? Thank you very much!
